samples/subsys/usb/cdc_acm/README.rst

```diff
@@ -0,0 +1,63 @@
+.. _usb_cdc_acm:
+
+USB CDC ACM Sample Application
+####################################
+
+Overview
+********
+
+Sample app for USB CDC ACM driver. The received data is echoed back
```

Comment:
This sample app demonstrates use of a USB Communication Device Class (CDC) Abstract Control Model (ACM) driver provided by the Zephyr project.  Received data is echoed back to the serial port provided by this driver.